docs/topics/native/native-spm.md

```diff
@@ -132,29 +164,32 @@ To set up the publishing of an XCFramework:
     
     The output will describe any errors found or show the successful download and parsing result if the manifest is correct.
 
-6. Push the `Package.swift` file to your remote repository. Make sure to create and push a Git tag with the
+7. Push the `Package.swift` file to your remote repository. Make sure to create and push a Git tag with the
    semantic version of the package.
 
-### Add the package dependency
+### Add a package dependency
 
-Now that both files are accessible, you can add the package dependency:
+Now that both files are accessible, you can add a package dependency to an existing client iOS project or create a new
+project. To add a package dependency:
```

Comment:
```suggestion
project. To add a dependency on the Swift package you created:
```

"The" here showed that we're adding a dependency specifically on the package we just released, not just any old package dependency. Should we be specific here in some way?

Reply:
Thank you! rephrased the intro + fixed the article